modules/dnn/test/test_model.cpp

```diff
@@ -202,6 +221,21 @@ TEST_P(Test_Model, DetectionMobilenetSSD)
                     scoreDiff, iouDiff, confThreshold, nmsThreshold, size, mean, scale);
 }
 
+TEST_P(Test_Model, Segmentation)
+{
+    std::string inp = _tf("boat.jpg");
+    Net network = readNetFromCaffe(_tf("fcn8s-heavy-pascal.prototxt"), _tf("fcn8s-heavy-pascal.caffemodel"));
+    std::string exp = _tf("segmentation_exp.png");
```

Comment:
Please rewrite test so it'll initialize Model from weights and prototxt inside `testSegmentationModel`. Check existing tests for reference.

Reply:
@dkurt, done